### PR TITLE
Fullscreen slides

### PR DESCRIPTION
**Pull request by ktbee**
We got full screen slides!!

Screen Shot 2019-11-01 at 1 47 43 PM

These changes:
- allow only one slide to be visible at a time in the frontend view of scenarios
- allows all slides to be visible in the editor preview
- advances slides on click in scenario run and not in preview

### CODE REVIEW

**Comment by ktbee** [comment]
Ugh actually I think there might be a little wonkiness to work out with the click handlers, but I'd love to know what you think so far!

**Comment by rwaldron** [comment]
Looks great, but something weird is going on... I viewed a moment, hit "Next" and it brought me to a different moment!

**Comment by ktbee** [comment]
Hmm I think I figured out why you were seeing another moment @rwaldron. I think previous slides were cached from viewing other moments. If you refresh the page while viewing a moment, that should "fix" it. I'm working on a more long term fix now!

**Comment by ktbee** [comment]
I think I've fixed the weirdness! Ready for another look, when y'all are ready.

File: client/components/Scenario/index.jsx [hunk 1]
```diff
@@ -13,87 +15,145 @@ class Scenario extends Component {
...
-            this.props.setSlides({ slides });
-        } else {
-            this.props.setSlides({ slides: null });
+            return slides;
         }
+        return null;
+    }
+
+    async getScenarioSlides() {
+        const cardClass = this.isScenarioRun()
```
Comment (evmiguel) [inline client/components/Scenario/index.jsx]: I like that the same slides list can be styled based on preview or run views

File: client/components/Scenario/index.jsx [hunk 2]
```diff
@@ -13,87 +15,145 @@ class Scenario extends Component {
...
+        );
+        const scenarioSlides = [];
+
+        scenarioSlides.push(descriptionSlide);
+        contentData.map((slide, index) => {
+            const isLastSlide = index === contentData.length - 1;
+            const slideButton = isLastSlide
+                ? this.getSlideButton('finish')
+                : this.getSlideButton('next');
+            const displaySlide = ContentSlide(slide, cardClass, slideButton);
```
Comment (evmiguel) [inline client/components/Scenario/index.jsx]: I like how the preview of a slide also shows a 'next' or 'finish' button

File: client/components/Scenario/index.jsx [hunk 3]
```diff
@@ -13,87 +15,145 @@ class Scenario extends Component {
...
     }
 
     render() {
-        const { title, description, slides } = this.props;
-        return (
+        const { slides } = this.props;
+        return this.isScenarioRun() ? (
             <Grid columns={1}>
                 <Grid.Column>
-                    <Grid.Row key="meta">
```
Comment (evmiguel) [inline client/components/Scenario/index.jsx]: Nice cleanup!

**Pull request merged**